build-system/tasks/init-bento-component/amp-__component_name_hyphenated__/validator-amp-__component_name_hyphenated__.protoascii

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
I think this is okay to keep but generally not essential. None of the Bento components have this file yet, and I expect when it does get added it would be some PRs down the road rather than in the first one.

Reply:
When Bento-ing an existing component this should be ignored in favor of the existing one, so hopefully it won't even come up all that much for a while. Will leave it for now though as you say